test/Splunk.OpenTelemetry.AutoInstrumentation.IntegrationTests/ServerTimingHeaderTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿// <copyright file="ServerTimingHeaderTests.cs" company="Splunk Inc.">
+// Copyright Splunk Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#if NET6_0_OR_GREATER
```

Comment:
```suggestion
#if !NETFRAMEWORK
```

it could be easier when updating .NET version support

Reply:
https://github.com/signalfx/splunk-otel-dotnet/pull/94/commits/f95edd8007d5284fcb0a6ebfece3fc222f19046c